doc/api/esm.md

```diff
@@ -278,7 +278,15 @@ These CommonJS variables are not available in ES modules.
 
 `require` can be imported into an ES module using [`module.createRequire()`][].
 
-An equivalent for `__filename` and `__dirname` is [`import.meta.url`][].
+An equivalent for variable `__filename` and `__dirname` can be created inside each file with 
```

Comment:
The linter is not happy with this. Can you please respect the limit of 80 chars per line?
Thank you.